db/sql/cassandra/examples/simple/main.go

```diff
@@ -44,28 +46,50 @@ func (entity *User) SchemaName() string {
 }
 
 func main() {
-	err := exampleKeyspace()
+	cfg, err := loadConfig()
+	if err != nil {
+		fmt.Println("failed - configuration ", err)
+		os.Exit(1)
+	}
+
+	session, err := cassandra.CreateSessionFromClientConfigAndKeyspace(cfg, "")
```

Comment:
Tooooooo long name :-) (we can probably avoid that just by avoiding second parameter)